src/ucp/api/ucp.h

```diff
@@ -999,6 +1000,12 @@ typedef struct ucp_params {
      * will override the number of endpoints set by @e estimated_num_ppn
      */
     size_t                             estimated_num_ppn;
+
+    /**
+     * The name is intended for identification of context during analysis of
+     * UCX-based application (e.g. FUSE-based run time monitoring).
+     */
+    char                               *context_name;
```

Comment:
The name is intended for identification of context during tracing and analysis of ...
const char *name